src/TransformerGo.cs

```diff
@@ -297,32 +284,32 @@ private void FixStutteringTypeNames(CodeModelGo cmg)
-                                        (exported is IModelType && (exported as IModelType).Name.FixedValue.StartsWith(cmg.Namespace, StringComparison.OrdinalIgnoreCase)) ||
-                                        (exported is Method && (exported as Method).Name.FixedValue.StartsWith(cmg.Namespace, StringComparison.OrdinalIgnoreCase)));
+                                        (exported is IModelType modelType && modelType.Name.FixedValue.StartsWith(cmg.Namespace, StringComparison.OrdinalIgnoreCase)) ||
+                                        (exported is Method method && method.Name.FixedValue.StartsWith(cmg.Namespace, StringComparison.OrdinalIgnoreCase)));
 
             if (stutteringTypes.Any())
             {
                 stutteringTypes.ForEach(exported =>
                     {
-                        var name = exported is IModelType
-                                        ? (exported as IModelType).Name
-                                        : (exported as Method).Name;
+                        var name = exported is IModelType type
+                                        ? type.Name
+                                        : ((Method)exported).Name;
 
                         name = name.Value.TrimPackageName(cmg.Namespace);
 
                         var nameInUse = exportedTypes
-                                            .Any(et => (et is IModelType && (et as IModelType).Name.Equals(name)) || (et is Method && (et as Method).Name.Equals(name)));
-                        if (exported is EnumType)
+                                            .Any(et => (et is IModelType modeltype && modeltype.Name.Equals(name)) || (et is Method methodType && methodType.Name.Equals(name)));
+                        if (exported is EnumType enumType)
                         {
-                            (exported as EnumType).Name.FixedValue = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Enum");
+                            enumType.Name.Value = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Enum");
                         }
-                        else if (exported is CompositeType)
+                        else if (exported is CompositeType compositeType)
                         {
-                            (exported as CompositeType).Name.FixedValue = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Type");
+                            compositeType.Name.Value = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Type");
                         }
-                        else if (exported is Method)
+                        else if (exported is Method methodType)
                         {
-                            (exported as Method).Name.FixedValue = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Method");
+                            methodType.Name.Value = CodeNamerGo.AttachTypeName(name, cmg.Namespace, nameInUse, "Method");
                         }

```

Comment:
Have you tried regenerating the SDK? the last time I touched this the generation changed a little

Reply:
Yes , see first comment, all looks good.